### PR TITLE
Update config-cheat-sheet.en-us.md

### PR DESCRIPTION
**Pull request by bkmgit**
Add more information on configuring URI hyperlink rendering for Markdown.
Partially addressing  #8478

### CODE REVIEW

File: docs/content/doc/advanced/config-cheat-sheet.en-us.md [hunk 1]
```diff
@@ -108,6 +108,8 @@ Values containing `#` or `;` must be quoted using `` ` `` or `"""`.
 ## Markdown (`markdown`)
 
 - `ENABLE_HARD_LINE_BREAK`: **false**: Enable Markdown's hard line break extension.
+- `CUSTOM_URL_SCHEMES`: : Use a comma separated list (http,https,ftp,git,svn) to indicate 
+  URI hyperlinks that should be rendered in Markdown. The default is only http and https.
```
Comment (guillep2k) [inline docs/content/doc/advanced/config-cheat-sheet.en-us.md]: It's not URI but URL in this case. Also, http and https are always rendered no matter what this setting says.

Comment (zeripath) [reply docs/content/doc/advanced/config-cheat-sheet.en-us.md]: I think this should be part of the pr that does the custom URL schemes assuming we haven't merged that yet

Comment (bkmgit) [reply docs/content/doc/advanced/config-cheat-sheet.en-us.md]: @guillep2k @zeripath Thanks for suggestions. Updated contents as suggested. Feel free to incorporate into pr for custom URL schemes.

**Review by sapk — APPROVED** [review]

File: docs/content/doc/advanced/config-cheat-sheet.en-us.md [hunk 2]
```diff
@@ -108,6 +108,9 @@ Values containing `#` or `;` must be quoted using `` ` `` or `"""`.
 ## Markdown (`markdown`)
 
 - `ENABLE_HARD_LINE_BREAK`: **false**: Enable Markdown's hard line break extension.
+- `CUSTOM_URL_SCHEMES`: : Use a comma separated list (http,https,ftp,git,svn) to indicate
```
Comment (guillep2k) [inline docs/content/doc/advanced/config-cheat-sheet.en-us.md]:
```suggestion
- `CUSTOM_URL_SCHEMES`: Use a comma separated list (ftp,git,svn) to indicate additional
```

**Review by guillep2k — APPROVED** [review]

**Comment by guillep2k** [comment]
Build failing due to #8474 unrelated to this PR.

**Comment by zeripath** [comment]
Make lg-tm work

**Pull request merged**